server/views/pages/offence/check-offence-answers.njk

```diff
@@ -75,7 +75,7 @@
                 convictionDate: offence.sentence.convictionDate | formatDate,
                 terrorRelated: offence.terrorRelated,
                 isSentenced: offence.sentence,
-                custodialSentenceLength: offence.sentence.periodLengths | findPeriodLengthByType("SENTENCE_LENGTH"),
+                custodialSentenceLength: offence.sentence.periodLengths | findPeriodLengthByType("SENTENCE_LENGTH") if courtAppearance.hasOverallSentenceLength === 'true',
```

Comment:
this still needs to be shown regardless of an overall sentence length doesn't it?

Reply:
It was causing an error trying to render this template. Should there be a default value like '_Not entered_' or something?

Reply:
it shouldn't show if there isn't a value for SENTENCE_LENGTH, what is the error? I thought the if statement https://github.com/ministryofjustice/hmpps-remand-and-sentencing/blob/main/server/utils/nunjucksSetup.ts#L110 managed a null period lengths?

Reply:
You're right, the fix was the `=== 'true'` I added to Line 24. Not this one. I've removed this and have pushed up